github/authorizations.go

```diff
@@ -318,3 +318,39 @@ func (s *AuthorizationsService) Revoke(clientID string, token string) (*Response
 
 	return s.client.Do(req, nil)
 }
+
+// Create an impersonation OAuth token.
+//
+// This required admin permissions. With the returned Authorization.Token you can e.g. create or
+// delete a users ssh public key. NOTE: creating a new token automatically revokes an existing one.
```

Comment:
`users` -> `user's`?

`ssh public key` should be `public SSH key` according to https://github.com/google/go-github/blob/0e3941cd1f14d0322165504974eeefab7e1ddc6d/github/users_keys.go#L10.